flixel/FlxObject.hx

```diff
@@ -540,6 +549,17 @@ class FlxObject extends FlxBasic
 	 * Overriding this will force a specific color to be used for debug rect.
 	 */
 	public var debugBoundingBoxColor:Null<Int> = null;
+	
+	public var debugColorScheme(default, null):DebugColorScheme;
+	
+	/**
+	 * Can be overriden for RenderBlit mode to re-render stuff with new colors
+	 */
+	public function setDebugColorScheme(debugColorScheme:DebugColorScheme)
```

Comment:
What you propose is the original way I've done it.
On native targets it works fine because it just gets the colors every frame and uses them.
On Flash (+ FlxTilemap) there is some update stuff to be done on each change.